HTML/Cosmic.html

```diff
@@ -0,0 +1,39 @@
+﻿<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <title>Cosmic — Keep Talking and Nobody Explodes Module</title>
+        <meta name="viewport" content="initial-scale=1">
```

Comment:
```suggestion
        <meta charset="utf-8">
        <meta name="viewport" content="initial-scale=1">
        <title>Cosmic — Keep Talking and Nobody Explodes Module</title>
```

Reply:
Are we changing all of the manuals to this format? It looks like the template manual was used here. 
```HTML
<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Module Name — Keep Talking and Nobody Explodes Module</title>
    <meta name="viewport" content="initial-scale=1">
    <link rel="stylesheet" type="text/css" href="css/font.css">
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
    <link rel="stylesheet" type="text/css" href="css/main.css">```

Reply:
Yes, all manuals should use this new format. That's an old version of the template manual. [The template manual has been updated to use the new format.](https://github.com/Timwi/KtaneContent/blob/master/More/Template%20Manual.zip)